BUILD.gn

```diff
@@ -677,6 +677,7 @@ source_set("electron_lib") {
   ]
   if (enable_electron_extensions) {
     sources += filenames.lib_sources_extensions
+    deps += [ "//chrome/browser/resources:component_extension_resources" ]
```

Comment:
NB. this includes some other stuff we don't care about, but it was easier than making a new .grd file and adding it to the resource_ids list.